apps/dcos_overlay/src/dcos_overlay_poller.erl

```diff
@@ -158,21 +159,47 @@ config_overlay(Overlay, State) ->
     maybe_create_vtep(Overlay, State),
     maybe_add_ip_rule(Overlay, State).
 
+create_vtep(VXLan, #state{netlink = Pid}) ->
```

Comment:
nit: could you please move this function below and rename it to `create_vtep_link`? I believe it would be easier to understand that code if you use the following order:

```
maybe_create_vtep() ->
maybe_create_vtep_link() ->
match_vtep_link() ->
match() ->
create_vtep_link() ->
create_vtep_addr() ->
```